__tests__/app.test.js

```diff
@@ -162,3 +162,79 @@ describe("GET /api/articles/:article_id/comments", () => {
 
 })
 
+describe('POST /api/articles/:article_id/comments', () => {
+	test('should return a status code of 201 and a new comment posted to the database', () => {
+		const newComment = {
+			username: 'icellusedkars',
+			body: 'Great article on how to build endpoints!'
+		}		
+		
+		return request(app)
+		.post('/api/articles/6/comments')
+		.send(newComment)
+		.expect(201)
+		.then(({ body }) => {
+			expect(body.comment).toBe('Great article on how to build endpoints!')
+		})
```

Comment:
I think you should also be checking that it returns a comment id and all the other properties we expect to be on a comment too :)